index.html

```diff
@@ -10,14 +10,21 @@
 
     <div class="header">
         <div>
+          <div class="nav-cover">
             <nav>
                 <div class="logo"><a href="">ECHEZONA</a></div>
                 <a href="#" class="toggle-button">
                     <svg class="bar" width="18" height="12" viewBox="0 0 18 12" fill="none" xmlns="http://www.w3.org/2000/svg">
                     <path fill-rule="evenodd" clip-rule="evenodd" d="M0 1C0 0.447715 0.447715 0 1 0H17C17.5523 0 18 0.447715 18 1C18 1.55228 17.5523 2 17 2H1C0.447715 2 0 1.55228 0 1ZM0 6C0 5.44772 0.447715 5 1 5L17 5C17.5523 5 18 5.44772 18 6C18 6.55229 17.5523 7 17 7L1 7C0.447715 7 0 6.55228 0 6ZM1 10C0.447715 10 0 10.4477 0 11C0 11.5523 0.447715 12 1 12H17C17.5523 12 18 11.5523 18 11C18 10.4477 17.5523 10 17 10H1Z" fill="#6070FF"/>
                     </svg>
                 </a>
+                <ul id="hamburger-menu">
+                  <li class="nav-links"><a href="#portfolio">Portfolio</a></li>
+                  <li class="nav-links"><a href="#about">About</a></li>
+                  <li class="nav-links"><a href="#contact">Contact</a></li>
+              </ul>
```

Comment:
- [ ] As suggested by the previous reviewer, I think it would be better to fix the desktop view version to match the Figma template.

Kindly try to adjust CSS ```background-image``` of this section to match the [figma template.](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=zK4ByMtV9lSehD19-0) 😎😎😎

- [ ]  Kindly as suggested by the reviewer, It would be better to check also the responsiveness of your web page kindly take a look of your page and make it more professional 

## Your design
![image](https://user-images.githubusercontent.com/102757126/212933325-b4b84465-ce3d-4a99-8bbb-b9d1f38014cb.png)

## Figma design
![image](https://user-images.githubusercontent.com/102757126/212933616-de314683-cd75-46a7-8c36-db0f6504a220.png)